src/Core/StringLocalizer.cs

```diff
@@ -1,31 +1,42 @@
-﻿using Microsoft.Extensions.Localization;
-using System;
+﻿using System;
 using System.Collections.Generic;
 using System.Globalization;
+using System.Linq;
+using Microsoft.Extensions.Localization;
 
 namespace Core
 {
-    public class StringLocalizer : IStringLocalizer
-    {
-        readonly Func<IReadOnlyDictionary<string, IReadOnlyDictionary<string, string>>> modelProvider;
+	public class StringLocalizer : IStringLocalizer
```

Comment:
Would be good to name this for what it used for localization. E.g. DictionaryStringLocalizer could work.